pahfit/base.py

```diff
@@ -248,7 +248,30 @@ def __init__(self, obs_x, obs_y, estimate_start=False, param_info=None, filename
             fixed={"tau_sil": att_info["amps_fixed"][0]},
         )
 
+<<<<<<< HEAD
     def plot(self, axs, x, y, yerr, model, scalefac_resid=2):
+=======
+        # add additional att components to the model if necessary
+        self.att_info = att_info
+        if att_info is not None:
+            for k in range(len(att_info["names"])):
+                if att_info["names"][k] != 'S07_att':  # Only loop through att components that can be parameterized
+                    self.model *= att_Drude1D(
+                        name=att_info["names"][k],
+                        tau=att_info["amps"][k],
+                        x_0=att_info["x_0"][k],
+                        fwhm=att_info["fwhms"][k],
+                        bounds={
+                            "tau": att_info["amps_limits"][k],
+                            "fwhm": att_info["fwhms_limits"][k],
+                        },
+                        fixed={
+                            "x_0": att_info["x_0_fixed"][k],
+                        },
+                    )
+
+    def plot(self, ax, x, y, model):
+>>>>>>> 86cd523 (PAHFIT_v2.1)
```

Comment:
A conflict during the rebase that wasn't fixed. Clearly we should have searched for these when we were pair-rebasing @ThomasSYLai !